src/apis/us/UserCommunicatorImpl.java

```diff
@@ -15,6 +19,27 @@ public class UserCommunicatorImpl implements UserCommunicatorHandler{
     
     //SEE THE UserCommHandler TO FIND OUT WHAT THESE DO!!!
     
+    // reads a file of integers seperated by commas and stores then in List<Integer> which it returns
+    public List<Integer> readFile(File file) {
```

Comment:
File reading functionality is already handled by DataStore, the Coordinator should just make a request to that.